src/main/java/io/vertx/ext/eventbus/bridge/tcp/BridgeEvent.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2014 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.ext.eventbus.bridge.tcp;
+
+import io.vertx.codegen.annotations.CacheReturn;
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.net.NetSocket;
+
+/**
+ * Represents an event that occurs on the event bus bridge.
+ * <p>
+ * Please consult the documentation for a full explanation.
+ *
+ * @author <a href="http://tfox.org">Tim Fox</a>
+ */
+@VertxGen
+public interface BridgeEvent extends Future<Boolean> {
```

Comment:
I think we should propose moving this class to `vertx-bridge-common` so all bridges can use it.